src/member/views.py

```diff
@@ -1449,8 +1450,10 @@ def get_success_url(self):
         )
 
 
-class ClientStatusSchedulerDeleteView(generic.DeleteView):
+class ClientStatusSchedulerDeleteView(
+        PermissionRequiredMixin, generic.DeleteView):
```

Comment:
I think it may be unavoidable in some cases, even with a higher limit. But 79 is not enough for a Django application, for sure.